### ISSUE TITLE
Soccer V Football

### ISSUE THREAD

**Issue by riannella** [iptc/sport-schema#195]
Is the Preferred British English term for "Football" really "Soccer" ??

https://cv.iptc.org/newscodes/mediatopic/20001065

See: 
https://en.wikipedia.org/wiki/Football_in_the_United_Kingdom
https://dictionary.cambridge.org/dictionary/english/soccer

**Comment by pauljkelly** [comment]
Yes, this goes back a long time. In my understanding, to completely disambiguate, the alternative term "soccer" was chosen and the North American variety given the qualifier "American". It follows from there that Canadian and Australian rules football are also explicitly identified in the taxonomy.

You probably know this already but for the sake of the thread here: "Soccer" actually has a strong pedigree in the UK. Aside from originating in the UK it features in "World Soccer" magazine and elsewhere:

https://en.wikipedia.org/wiki/World_Soccer_(magazine)
https://www.etsy.com/ca/listing/1669086560/george-best-the-inside-story-vintage

@tfgrahame at actually collected examples when BBC was adopting SportsML

**Comment by riannella** [comment]
I can understand the history of the term soccer (we used to use it here in Australia too...)

But even the "World Soccer" wikipedia page starts off with "World Soccer is an English-language football magazine"
(ie it is about football)

As an international vocabulary, the influence should be from the common majority of the term usage.
The list of national football associations: https://en.wikipedia.org/wiki/Geography_of_association_football
has only 13 use the term Soccer, and the other 200+ use Football.

The good thing about SKOS is the use of multi-lingual preferred labels. 
So, the labels should align to the country/languages preferred usage. 
Alt labels can also be used.

So , for example, I would expect to see 
  skos:prefLabel "Football"@en-au
  skos:altLabel "Soccer"@en-au

**Comment by pauljkelly** [comment]
I'll refer this to the newscodes group since all this is really their jurisdiction.
